documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,294 @@
+
+The `instance type preferences` have replaced {ocp-name} templates. {project-short} currently falls back to using {ocp-name} templates when a relevant preference is not available.
+
+Customer Virtual Machine Mapping are custom templates that you can configure for specific migrations, VMs, and users.
+
+.Full support for migration from OVA
+
+{project-short} supports migration from OVA.
+
+.Integration of the `Create` and `Edit` migration plans
+
+The user interface `Create` and `Edit` elements for migration plans now align with the look and feel of the {ocp} web console and displays up-to-date data. These elements now replace Forms and Wizards. link:https://issues.redhat.com/browse/MTV-615[(MTV-615)]
+
+.VM posted with `Running` state
+
+{project-short} shows each VM's migration `Running` state during the migration. This single `Ronning` operation replaces two previous operations, `Create/Post VM` and Run VM`.  link:https://issues.redhat.com/browse/MTV-794[(MTV-794)]
+
+.Can download `must-gather` logs only using the CLI
+
+The {product-short} web console can no longer download logs. Every `must-gather` log image download must be preformed using CLI commands. See link:https://github.com/openshift/must-gather-operator[Must Gather Operator] for more information.
+
+.Review and configure migrated VMs
+
+The {project-short} web console shows the name and migration status of each VM. The migrated VMs are labeled `app: <vmname>` to help further changes in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-843[(MTV-843)]
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.Validation for RHEL 6
+
+Red Hat Enterprise Linux (RHEL) 9 does not support RHEL 6 as a guest operating system, and therefore RHEL 6 is not supported in {ocp-name} Virtualization. A validation of RHEL 6 guest operating system was added. link:https://issues.redhat.com/browse/MTV-413[(MTV413)]
+
+.Block migrations from {ocp-name} versions before 4.13
+
+The `Export API` in {ocp-name} Virtualization exposes VM manifests from {ocp-name} 4.13, so source {ocp} clusters must be version 4.13 or later. link:https://issues.redhat.com/browse/MTV-734[(MTV-734)]
+
+.Source and storage endpoints are not selected automatically
+
+Source network and source storage are not selected automatically when creating an {ocp} to {ocp} plan. link:https://issues.redhat.com/browse/MTV-729[MTV729]
+
+.Restored vSphere Verify certificate functionality
+
+The vSphere Verify certificate option in the `add-provider` dialog has been restored. It was removed in the transition to {ocp} console. link:https://issues.redhat.com/browse/MTV-737[(MTV-737)]
+
+.{ocp-name} Console native SDK fetch, post, and delete calls
+
+The `fetch`, `post`, and `delete` calls in the Forklift UI are now native to the {ocp-name} API. link:https://issues.redhat.com/browse/MTV-365[(MTV-365)]
```

Comment:
```suggestion
The `fetch`, `post`, and `delete` calls in the Forklift UI are now native to the {ocp-name} API. link:https://issues.redhat.com/browse/MTV-365[(MTV-365)]
```
Any more details that the user would benefit from?

Reply:
Query sent to Arik